alpine-consul/root/etc/services.d/consul-template/run

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/with-contenv sh
```

Comment:
@smebberson I haven't been clear. It does retry but problem is that it
logs an error. So what I'm trying to avoid is that error from happening.
Nothing crashes and consul-template does continue to work correctly.
On Dec 16, 2015 4:22 PM, "Scott Mebberson" <notifications@github.com> wrote:

> In alpine-consul/root/etc/services.d/consul-template/run
> <https://github.com/smebberson/docker-alpine/pull/13#discussion_r47854240>
> :
>
> > @@ -0,0 +1,23 @@
> > +#!/usr/bin/with-contenv sh
>
> @matthewvalimaki <https://github.com/matthewvalimaki>, I was reading the
> consul-template documentation a little more (I haven't actually used it
> before). Could we make use of the retry option? Would that cause
> consul-template not to error? Perhaps then we could remove the Consul
> nodes watch?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/smebberson/docker-alpine/pull/13/files#r47854240>.
>